rke/k8s_rke_system_images.go

```diff
@@ -8990,6 +8990,47 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.21.1",
 		},
+		"v1.24.17-rancher1-2": {
```

Comment:
Since `v1.24.17-rancher1-1` is not released yet (and will be released as part of August k8s patches release) please update versions on it instead (as well as change version ranges to apply new templates from `v1.24.17-rancher1-1`).
We don't release 2 versions at the same time generally - there is no point of releasing `v1.24.17-rancher1-1` without ACI bump and `v1.24.17-rancher1-2` with ACI bump.

Reply:
Made requested changes: updated versions on `v1.24.17-rancher1-1`  and changed version ranges.